Marlin/src/pins/stm32f1/pins_BTT_SKR_E3_DIP.h

```diff
@@ -25,7 +25,7 @@
   #error "Oops! Select an STM32F1 board in 'Tools > Board.'"
 #endif
 
-#define BOARD_INFO_NAME "BIGTREE SKR E3 DIP V1.0"
+#define BOARD_INFO_NAME "BTT SKR E3 DIP V1.0"
```

Comment:
For consistency with the other E3 Mini boards, I changed this to "BTT SKR Mini E3 DIP V1.0". While longer, it's still under the 25 character limit.